topsort-endpoints.yml

```
@@ -155,6 +186,11 @@ components:
           type: string
           description: The marketplace's ID for the product. It will match the ID for a product in the request's product array.
           example: pSA0238
+        assertUrl:
```

I'm not sure about the case here (assetURL or assetUrl). @fcifuentesq can you confirm?
```suggestion
        assetUrl:
```

---

fwiw, the JSON field should be assetUrl (like productId, someHtmlRedirect, etc) according to our naming conventions in notion guidelines. In Go: assetURL (productID, someHTMLRedirect).